app/plugins/runscope.md

```diff
@@ -0,0 +1,66 @@
+---
+id: page-plugin
+title: Plugins - Runscope
+header_title: Runscope
+header_icon: /assets/images/icons/plugins/runscope.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Getting Started
+    items:
+      - label: Installation
+      - label: Configuration
+---
+
+Logs request and response data to [Runscope][runscope]. Using the Runscope Traffic Inspector, each API call can be fully viewed in it's entirety. All traffic can be searched by keyword (headers and bodies are indexed) and attribute (i.e. status code, response size, response time, etc.). Using Runscope [Live Traffic Alerts][live-traffic-alerts], API failures and exceptions can be caught, notifying your team about problems before your customers find out. Trigger alerts based on any part of the HTTP request or resopnse, including header values, JSON or XML data, connection details and more. Alerts can be sent to Slack, HipChat, PagerDuty, email, or webhook notifications. Live Traffic Alerts is available on all medium and larger plans.
+
+----
+
+## Installation
+
+Add the plugin to the list of available plugins on every Kong server in your cluster by editing the [kong.yml][configuration] configuration file:
+
+```yaml
+plugins_available:
+  - runscope
+```
+
+Every node in the Kong cluster must have the same `plugins_available` property value.
```

Comment:
This step/chapter is not required anymore for 0.6.0. Plugins included in Kong do not have to be added in the configuration anymore. The current documentation for other plugins reflect those changes.